### ISSUE TITLE
cif_core: _atom_site.refinement_flags should be deprecated in a machine readable way

### ISSUE THREAD

**Issue by vaitkus** [COMCIFS/cif_core#104]
The `_atom_site.refinement_flags` data item has been deprecated and split into three distinct data items since the later versions of `DDL1`. However, currently the deprecation status is only reflected in the  human readable definition of this data item and not by using the `_definition.replaced_by` data item.

I can see that there is a slight problem since the `_definition.replaced_by` data item only allows a single replacement tag whilst in this case three would be preferred  (`_atom_site.refinement_flags_posn`, `_atom_site.refinement_flags_adp`, `_atom_site.refinement_flags_occupancy`). However, since the `_definition.replaced_by` data item is a recent development on itself, maybe redefining it as a part of a new looped category would be possible?

**Comment by jamesrhester** [comment]
It seems clear that there is a need for this to be looped. I will run it by the DDLm group.

**Comment by jamesrhester** [comment]
Here is a proposed replacement:

```
save_DEFINITION_REPLACED

    _definition.id               DEFINITION_REPLACED
    _definition.scope            Category
    _definition.class            Loop
    _definition.update           2019-01-08
    _description.text
;
     Attributes used to describe deprecated and replaced definitions
;
    _name.category_id            DEFINITION
    _name.object_id              DEFINITION_REPLACED
    _category_key.name           '_definition_replaced.id'


save_

save_definition_replaced.by

    _definition.id               '_definition_replaced.by'
    _definition.class            Attribute
    _definition.update           2019-01-08
    _description.text
;
     A dataname that should be used instead of the defined dataname.
     The defined dataname is deprecated and should not be used.
;
    _name.category_id            definition_replaced
    _name.object_id              by
    _type.container              Single
    _type.purpose                Encode
    _type.contents               Tag
    _type.source                 Assigned

save_

save_definition_replaced.id

    _definition.id               '_definition_replaced.id'
    _definition.class            Attribute
    _definition.update           2019-01-08
    _description.text
;
     An opaque identifier for the replacement
;
    _name.category_id            definition_replaced
    _name.object_id              id
    _type.purpose                Key
    _type.source                 Assigned
    _type.container              Single
    _type.contents               Code

save_

```

**Comment by jamesrhester** [comment]
The DDLm group has raised no objections to this proposal, so these new definitions can be incorporated into DDLm.

**Comment by vaitkus** [comment]
Changes have been implemented in the PR #126.

**Comment by jamesrhester** [comment]
Thanks for submitting the PR.
